react/tic-tac-toe/src/app/screens/GameHistory/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+
+import styles from './styles.module.scss';
+
+import MatchesService from '~services/MatchesService';
+
+const Spinner = require('react-spinkit');
+
+class GameHistory extends Component {
+  state = {
+    isLoading: true,
+    history: [],
+    error: ''
+  };
+
+  getData() {
```

Comment:
Define methods as arrow functions, except for lifecycle ones. 
https://github.com/Wolox/tech-guides/blob/tech-refactor/frontend/react/style-guide.md#methods
This is so the `this` reference points to the component and not the function.